daq_macros.py

```diff
@@ -2563,27 +2563,33 @@ def vectorZebraScanNormal(vecRequest):
   
   sweep_start_angle = reqObj["sweep_start"]
   sweep_end_angle = reqObj["sweep_end"]
+  range_degrees = int(reqObj["sweep_end"]) - int(reqObj["sweep_start"])
   imgWidth = reqObj["img_width"]
   expTime = reqObj["exposure_time"]
   numImages = int((sweep_end_angle - sweep_start_angle) / imgWidth)
-  x_vec_start=reqObj["vectorParams"]["vecStart"]["x"]
-  y_vec_start=reqObj["vectorParams"]["vecStart"]["y"]
-  z_vec_start=reqObj["vectorParams"]["vecStart"]["z"]
-  x_vec_end=reqObj["vectorParams"]["vecEnd"]["x"]
-  y_vec_end=reqObj["vectorParams"]["vecEnd"]["y"]
-  z_vec_end=reqObj["vectorParams"]["vecEnd"]["z"]
-  setPvDesc("vectorStartOmega",sweep_start_angle)
-  setPvDesc("vectorEndOmega",sweep_end_angle)  
-  setPvDesc("vectorStartX",x_vec_start)
-  setPvDesc("vectorStartY",y_vec_start)  
-  setPvDesc("vectorStartZ",z_vec_start)  
-  setPvDesc("vectorEndX",x_vec_end)
-  setPvDesc("vectorEndY",y_vec_end)  
-  setPvDesc("vectorEndZ",z_vec_end)  
-  setPvDesc("vectorframeExptime",expTime*1000.0)
-  setPvDesc("vectorNumFrames",numImages)
   scanWidth = float(numImages)*imgWidth
```

Comment:
one of either range_degrees or scanWidth should be used - they are the same value and used in either the call to zebraDaq() or zebraDaqBluesky()

for code similarity, I would suggest removing range_degrees - although that is more intuitive code than scanWidth

Reply:
removed range_degrees in 3fbdaa8